File: ru.kmorozov.api.onedrive/src/main/java/ru/kmorozov/onedrive/client/exceptions/OneDriveErrorInfo.java
package ru.kmorozov.onedrive.client.exceptions;

import com.google.api.client.util.Key;

/**
 * Created by sbt-morozov-kv on 13.03.2017.
 */
public class OneDriveErrorInfo {

    @Key("error")
    String error;
    @Key("error_description")
    String description;

    public String getError() {
        return this.error;
    }

    public void setError(String error) {
        this.error = error;
    }

    public String getDescription() {
        return this.description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    @Override
    public String toString() {
        return "OneDriveErrorInfo{" +
                "error='" + this.error + '\'' +
                ", description='" + this.description + '\'' +
                '}';
    }
}

File: ru.kmorozov.api.onedrive/src/main/java/ru/kmorozov/onedrive/client/exceptions/OneDriveErrorInfo.kt
package ru.kmorozov.onedrive.client.exceptions

import com.google.api.client.util.Key

/**
 * Created by sbt-morozov-kv on 13.03.2017.
 */
class OneDriveErrorInfo {

    @Key("error")
    lateinit var error: String
    @Key("error_description")
    var description: String = ""

    override fun toString(): String {
        return "OneDriveErrorInfo{" +
                "error='" + error + '\''.toString() +
                ", description='" + description + '\''.toString() +
                '}'.toString()
    }
}
